subsys/net/openthread/rpc/server/ot_rpc_commissioning_server.c

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <ot_rpc_ids.h>
+#include <ot_rpc_types.h>
+#include <ot_rpc_common.h>
+
+#include <nrf_rpc/nrf_rpc_serialize.h>
+#include <nrf_rpc/nrf_rpc_cbkproxy.h>
+
+#include <zephyr/net/openthread.h>
+
+#include <openthread/link.h>
+
+static void ot_thread_discover_cb(otActiveScanResult *result, void *context, uint32_t callback_slot)
+{
+	struct nrf_rpc_cbor_ctx ctx;
+	struct ot_rpc_active_scan_result pkt_data;
+	size_t cbor_buffer_size = result ? sizeof(pkt_data) + 1 : 1;
```

Comment:
```suggestion
	size_t cbor_buffer_size = result ? sizeof(pkt_data) + 2 : 1;
```
Extra byte for encoding the length.

Reply:
was redesigned. not actual anymore